setup.py

```diff
@@ -32,7 +32,7 @@
     "scipy>=1.0.0,<2",
     "tqdm>=4.31.1,<5",
     "matplotlib>=2.2.2,<4",
-    "scikit-learn>=0.22.1,<1",
+    "scikit-learn>=0.22.1,<1.0.3",
```

Comment:
is there any breaking change between <1 and >1?

Reply:
Yes, mainly Python < 3.7 isn't supported. But we still have `scikit-learn>=0.22.1`, so Python 3.6 is ok.
See https://pypi.org/project/scikit-learn/1.0/

_Scikit-learn 0.20 was the last version to support Python 2.7 and Python 3.4. scikit-learn 0.23 and later require Python 3.6 or newer. scikit-learn 1.0 and later require Python 3.7 or newer._